src/services/business/update-business-email-change-service.js

```diff
@@ -1,15 +1,15 @@
 import { fetchBusinessDetailsService } from './fetch-business-details-service.js'
 import { flashNotification } from '../../utils/notifications/flash-notification.js'
 
-const updateBusinessEmailChangeService = async (yar) => {
-  const businessDetails = await fetchBusinessDetailsService(yar)
+const updateBusinessEmailChangeService = async (yar, credentials) => {
+  const businessDetails = await fetchBusinessDetailsService(yar, credentials)
 
   businessDetails.contact.email = businessDetails.changeBusinessEmail
   delete businessDetails.changeBusinessEmail
 
   yar.set('businessDetails', businessDetails)
 
-  flashNotification(yar, 'Success', 'You have updated your business email address')
+  flashNotification(yar, 'Success', 'You have updated your business email')
```

Comment:
This success message was fixed to be 'You have updated your business email address' (as per our prototype), but now it's been reverted to 'You have updated your business email', which is incorrect

[Prototype link](https://fcp-farming-front-door-59e0a6ac6fb8.herokuapp.com/MVP/details-business-details?changed=true&emailchanged=true)